### ISSUE TITLE
KTXProcessor.convert to KTX file create not valid image

### ISSUE THREAD

**Issue by Nicolas02** [libgdx/libgdx#5703]
#### Issue details

I try to convert png file to ktx file with KTXProcessor.convert

``` java
	GdxNativesLoader.load();
	KTXProcessor.convert(sourcePngFile, ktxFile1, false, true, false);
```
OR
``` java
	GdxNativesLoader.load();
	KTXProcessor.main((String[]) Arrays.asList(sourcePngFile, ktxFile2, "-etc1").toArray());
```

The code works with no error 

``` java 
0 = original_1024.png
1 = original_1.ktx
2 = -etc1
Processing :original_1024.png for face #0
Level: 0
Finished

Process finished with exit code 0
```

But when I use PVRTexTool to visualize the file, there is the following message : 
"Either file is not a valid image or texture file, or is an unsupported format. If the file is a simple bitmap or texture file, you could try wrapping it as raw data.

#### Version of LibGDX and/or relevant dependencies
Version 1.9.9

#### Please select the affected platforms
- [ ] Android
- [ ] iOS (robovm)
- [ ] iOS (MOE)
- [ ] HTML/GWT
- [X ] Windows
- [ ] Linux
- [ ] MacOS

**Comment by MrStahlfelge** [comment]
KTX is not supported in this repo, please add an issue on the KTX issue tracker.

**Comment by tommyettinger** [comment]
The reason for the confusion is that there's a separate library called libKTX that uses libGDX but is unrelated to the KTX image format. I can look into this, maybe, but not using libGDX 1.9.9... that's over a year and a half behind the current version.

**Comment by tommyettinger** [comment]
OK, I investigated, and I haven't learned much other than:
  - KTXProcessor can generate .ktx files fine, but...
  - PVRTexTool can't read them, though...
  - libGDX can read them, however...
  - The Texture class can load .ktx files, but the Pixmap class cannot.
  - KTX files have to be power-of-two size to be loaded (and maybe also square to be shown correctly?)
  - I'm not sure how to get ETC1a to show alpha in a Texture.

So in short, this isn't really a libGDX issue, because libGDX can use the KTX files it produces, even if PVRTexTool can't. There could be some missing info in the KTX files that PVRTexTool needs, but since we don't know what PVRTexTool is having trouble with, I'm not sure what we can do. You can use libGDX to load KTX files fine, and if there is a flaw in the generated files, it only affects external tools. I'm closing this.
